src/biogeochem/EDBGCDynMod.F90

```diff
@@ -266,7 +269,7 @@ subroutine EDBGCDynSummary(bounds, num_soilc, filter_soilc, num_soilp, filter_so
        c13_soilbiogeochem_carbonflux_inst, c13_soilbiogeochem_carbonstate_inst, &
        c14_soilbiogeochem_carbonflux_inst, c14_soilbiogeochem_carbonstate_inst, &
        soilbiogeochem_nitrogenflux_inst, soilbiogeochem_nitrogenstate_inst, &
-       clm_fates, nc)
+       nc)
```

Comment:
Elsewhere I added clm_fates as an argument to subroutine calls, ultimately for use by MIMICS. Here, however, I removed it because it seemed unused. Let me know if I shouldn't have.

Reply:
If it's not currently used you should remove it.